frontend/src/pages/AboutPage.js

```diff
@@ -145,10 +145,10 @@ function AboutPage() {
 
       <TextH1>Database Refresh</TextH1>
       <TextBody>Will refresh the database with the latest data from the N3C Enclave.</TextBody>
-      <TextBody><b>IMPORTANT:</b> Concept set members are currently slowly updated in the API. As a tentative solution,
-        to prevent bugs, new containers and code sets will not be imported into TermHub until those members are also
-        available for fetching. This unfortunately slows down fetching of new code sets from being otherwise
-        instantaneous to hours or days.</TextBody>
+      <TextBody><b>IMPORTANT:</b> There is a delay in the Enclave where when a concept set draft is finalized, its
```

Comment:
## About page updates
@Sigfried (CC @stephanieshong @hlehmann17)
This pull request handles the issue where, when a new concept set version is finalized, its concept set members are not available for up to 45 minutes. I added a note about this for reference on the About page next to the refresh button.

I'd also like to eventually display additional info, such as if a refresh is active, how long it's been going for, and if any special new csets are being fetched. I created an issue for that:
- #498